samples/package.json

```diff
@@ -8,15 +8,28 @@
     "node": ">=8"
   },
   "repository": "googleapis/nodejs-asset",
-  "private": true,
   "nyc": {
     "exclude": [
       "**/*.test.js"
     ]
   },
-  "scripts": {},
+  "scripts": {
+    "test": "repo-tools test run --cmd ava -- -T 20s --verbose system-test/*.test.js"
```

Comment:
Please just do `mocha system-test` here

Reply:
Never used mocha before, so this line should be like below?
"test": "repo-tools test run --cmd mocha system-test"